src/js/src/components/ToolboxComponents/DomainStats.vue

```diff
@@ -30,19 +30,19 @@
           </thead>
           <tbody>
             <tr>
-              <td>Size in KB</td>
+              <td v-html="$t('toolBox:size_in_kb')"></td>
               <td v-for="(item, index) in rawData" :key="index">
                 {{ item.sizeInKb.toLocaleString("en") }}
               </td>
             </tr>
             <tr>
-              <td>Total pages</td>
+              <td v-html="$t('toolBox:total_pages')"></td>
               <td v-for="(item, index) in rawData" :key="index">
                 {{ item.totalPages.toLocaleString("en") }}
               </td>
             </tr>
             <tr>
-              <td>Ingoing links</td>
+              <td v-html="$t('toolBox:ingoing_links')"></td>
```

Comment:
Consider using the standard Vue `<td>{{$t('toolBox:total_pages')}}</td>` instead of the v-html directive